openshift/si-bandit-template.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: adviser
+  annotations:
+    description: "Thoth: Adviser"
+    openshift.io/display-name: "Thoth: Adviser"
+    version: 0.6.4
+    tags: thoth,ai-stacks,adviser
+    template.openshift.io/documentation-url: https://github.com/Thoth-Station/
+    template.openshift.io/long-description: >
+      This template defines resources needed to run recommendation logic of Thoth to OpenShift.
+    template.openshift.io/provider-display-name: "Red Hat, Inc."
+  labels:
+    app: thoth
+    template: adviser
+    component: adviser
+
+parameters:
+  - name: THOTH_SI_BANDIT_JOB_ID
+    required: true
+    description: A unique dentifier of bandit security indicator job.
+    displayName: si-bandit id
+  - name: THOTH_SI_BANDIT_PACKAGE_NAME
+    required: true
+    description: Name of package which is being analyzed.
+    displayName: package name
+  - name: THOTH_SI_BANDIT_PACKAGE_VERSION
+    required: true
+    description: Version of the package being analyzed.
+    displayName: package version
+  - name: THOTH_SI_BANDIT_PACKAGE_INDEX
+    required: true
+    description: Index to get package from.
+    displayName: package Index
+    value: "https://pypi.org/simple"
+
+objects:
+  - apiVersion: argoproj.io/v1alpha1
+    kind: Workflow
+    metadata:
+      name: "${THOTH_SI_BANDIT_JOB_ID}"
+      annotations:
+        thoth-station.ninja/template-version: 0.6.4
+      labels:
+        app: thoth
+        component: si-bandit
+    spec:
+      serviceAccountName: argo
+      ttlStrategy:
+        # Keep adviser 1h to let users browse the logs.
```

Comment:
this logic might not be required. have a look at https://github.com/argoproj/argo/blob/master/examples/pod-gc-strategy.yaml and https://github.com/argoproj/argo/blob/master/examples/gc-ttl.yaml, but in general we want pods and workflows to be deleted if succesfull